src/Job.php

```diff
@@ -85,6 +90,26 @@ public function delete(): void
         $this->consumer->acknowledge($this->message);
     }
 
+    /**
+     * @param mixed $data
+     */
+    public function setMessageContext($data): void
+    {
+        $this->message->setProperty(static::CONTEXT_PROPERTY, \serialize($data));
+    }
+
+    /**
+     * @return mixed|null
+     */
+    public function getMessageContext()
```

Comment:
add support for some default value, like `$default = null`

Reply:
+ check does vendor has method like `hasProperty`

Reply:
no `hasProperty`, `$default` added